src/proxy-page/proxy-page.service.ts

```diff
@@ -52,7 +52,7 @@ export class ProxyPageService {
   ) {
     this.context.Init(spaceKey, pageId, theme, style);
     this.context.setTitle(results.title);
-    this.context.setHtmlBody(results.body.styled_view.value);
+    this.context.setHtmlBody(results.body.view.value);
```

Comment:
Great!

It worked almost first time for me, and if we incorporate my other suggestion about the setHtmlBody function we can use the same code for loading page HTML whether it's for the page-proxy or page API.

Reply:
Just realised that the `initContext` function would also need updating as that's the one that calls `setHtmlBody` . In the new page API code I have currently changed that to a central function as it is duplicated between the page API page-proxy services but it would still be pretty simple to handle that logic.